src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,171 @@
+All versions of CockroachDB released after November 18, 2024, including patch fixes for prior versions 23.1 onward, are made available under the [CockroachDB Software License][csl]. You can use the dropdown menu at the top of the page to view documentation for other versions.
+
+<a name="types-of-licenses"></a>
+
+## License options
+
+The table of licenses below refers to options for {{ site.data.products.core }} deployments. All {{ site.data.products.cloud }} deployments automatically have a valid **Enterprise** license.
+
+Type | Description
+-------------|------------
+**Enterprise** <a name="enterprise"></a> | This paid license allows usage of all CockroachDB features in accordance with the terms specified in the [CockroachDB Software License][csl]. License must be renewed annually or as negotiated. Support levels available include [Enterprise][support] or [Essential][support].
+**Enterprise Free** <a name="enterprise-free"></a> | Same functionality as **Enterprise**, but free of charge for businesses with less than $10M in annual revenue. Clusters will be [throttled](#throttling) after 7 days without sending telemetry. License must be renewed annually. Support level available is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
+**Enterprise Trial** <a name="enterprise-trial"></a> | A 30 day self-service trial license. Telemetry is required during the trial. Clusters will be [throttled](#throttling) after 7 days without sending telemetry. Telemetry can be disabled once the cluster is upgraded to a paid **Enterprise** license. Support level available during trials is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
+
+<a href="mailto:sales@cockroachlabs.com">Contact Sales</a> if you want to try CockroachDB without telemetry requirements or if you require an extended trial period.
+
+{{site.data.alerts.callout_success}}
+Note that:
+- Clusters with no license key have a 7-day grace period before requiring a license key. This is useful for ephemeral development clusters.
```

Comment:
Need a line break above this; the bullets aren't rendering

Reply:
ARRRRRGGGGHHHHH MARKDOWNNNNNNNNNNN

✅